hibernate-core/src/main/java/org/hibernate/loader/ast/internal/LoaderSelectBuilder.java

```diff
@@ -716,7 +715,7 @@ private void applyOrdering(TableGroup tableGroup, OrderByFragment orderByFragmen
 				// 'entity graph' takes precedence over 'fetch profile'
 				if ( entityGraphTraversalState != null ) {
 					traversalResult = entityGraphTraversalState.traverse( fetchParent, fetchable, isKeyFetchable );
-					fetchTiming = traversalResult.getFetchStrategy();
```

Comment:
This method name is confusing esp. when we have a FetchStrategy class already. I changed it to better reflect its purpose.